android/app/src/main/java/org/fossasia/openevent/activities/SearchActivity.java

```diff
@@ -232,5 +238,10 @@ public void handleVisibility() {
             searchRecyclerView.setVisibility(View.VISIBLE);
         }
     }
+
+    @Override
+    public void showSnackbar(int actionCode, SnackbarHelper snackbarHelper) {
```

Comment:
Encapsulate `actionCode` inside object itself. Change int to enum, rename SnackbarHelper to `BookmarkStatus`